src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -275,14 +277,27 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
                     });
                 }
 
+                if (string.Equals(_request.Project.RestoreMetadata?.RestoreAuditProperties?.EnableAudit, "enable", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
@JonDouglas Nuget already has a bunch of MSBuild properties which use `true` and `false` for boolean values.  I kept `enable` here, since that's what the spec called for, but I just wanted to call it out. While implementing this, I'm wondering if it has implications on project systems that want to make a properties page to change the value. They probably already have controls for true/false, so we might be causing them extra work if we want to keep "enable", in addition to any possible customer confusion about why this is the only "boolean like" property that doesn't use "true".

Reply:
This is likely oversight on my part. When I wrote that, I was looking at a minimal sdk-style project and the only new features being enabled had this `enable/disable` usage. Upon looking at https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props I believe 99% of properties use true/enable.

Let's do what is most consistent.